airflow/contrib/executors/kubernetes_executor.py

```diff
@@ -424,13 +459,11 @@ def _make_kube_watcher(self):
         watcher.start()
         return watcher
 
-    def _health_check_kube_watcher(self):
-        if self.kube_watcher.is_alive():
-            pass
-        else:
+    def _ensure_kube_watcher(self):
+        if not self.kube_watcher.is_alive():
+            # TODO: #PROMETHEUS increment counter
```

Comment:
Number of watcher restarts is a good candidate for a metric. If prometheus integration is on the horizon, I think it is wise to start tagging location.

Reply:
We can add a `statsd` counter until then.

Reply:
```suggestion
            Stats.incr("executor.kube_watcher.restarts")
```

Reply:
perfect! thank you